types/ckeditor__ckeditor5-utils/src/keystrokehandler.d.ts

```diff
@@ -1,6 +1,6 @@
-import Emitter from "./emittermixin";
-import { KeystrokeInfo } from "./keyboard";
+import { Emitter } from "./emittermixin";
 import { PriorityString } from "./priorities";
+import * as ckEngine from "@ckeditor/ckeditor5-engine";
```

Comment:
Since `utils` v27 depends on `engine`, and then `engine` depends back on `utils`, I can't make `engine` depend on an older version of `utils`. This made both `engine` and `core` need to be updated to require TS 4.2.